src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4385 @@
+
+In the **Hierarchy** delete the object **Default Floor** as we will not be needing this for the track.
+
+![!Remove Default Floor](../img/RaceTimerTutorial/remove_default_floor.png){: .center loading="lazy" }
+
+We now need to create a track for players to run on. A track has been uploaded to **Community Content** to help you get started. This can be imported from the **Community Content** panel by searching for **Race Timer Tutorial - Track** by **CommanderFoo**.
+
+![!Community Content Template](../img/RaceTimerTutorial/import_track.png){: .center loading="lazy" }
+
+Drag and drop the template **Race Timer Tutorial - Track** into the **Hierarchy** and **Deinstance** it.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/cc_template_deinstance.mp4" type="video/mp4" />
+    </video>
+</div>
+
+If you enter **Play** mode you will notice the spawn position isn't in an ideal location for any player that joins the game. Move the **Spawn Point** so that it's before the starting position on the track.
+
+!!! tip
+    Pressing ++V++ will toggle the visibility of Gizmos. With gizmo visibility on, it's easy to see where to move objects such as the **Spawn Point**.
+
+![!Spawn Point](../img/RaceTimerTutorial/move_spawn_point.png){: .center loading="lazy" }
+
+## Starting Line
+
+When a race is about to start, we need to move the players to the starting line. Instead of grouping all players up in one spot, we can create a few positions along the starting line and randomly pick one to place the player at. There is still a possibility of players spawning in the same location, and to fix this it would require a more advanced system to keep track of which lanes have been assigned. For this tutorial we will keep it simple.
+
+### Creating Lane Positions
+
+!!! tip
+    As a visual aid to help move the starting positions, an object such as a **Cube** can be used. For performance it's recommended to use a **Group** when needing to reference a position for spawning in objects. **Group** objects have minimal properties (i.e. no **Rendering**) compared to other objects.
+
+1. Create a new group called **Starting Positions** inside the **Race Timer Tutorial - Track** group.
+2. Drag and drop a **Cube** from **Core Content** into the **Starting Positions** group.
+3. Set **Game Collision** to **Force Off** for the **Cube** object.
+4. Set **Camera Collision** to **Force Off** for the **Cube** object.
+5. Rename **Cube** to **Position**.
+6. Duplicate and space these out along the starting line.
+7. Set **Visibility** to **Force Off** for all **Position** objects.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/create_start_positions.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Setup Script Hierarchy
```

Comment:
Rename this section Client and Server context, and creating another h3 with this title just under it. 

Setup --> Set up